exchange/cache.go

```diff
@@ -309,6 +329,8 @@ func DeleteCacheResourceFromChange(change ExchangeChange, nodeId string) {
 	} else if change.IsServicePolicy() {
 		id, arch, vers := svcInformationFromSvcId(change.ID)
 		DeleteCacheResource(SVC_POL_TYPE_CACHE, ServicePolicyCacheMapKey(change.OrgID, id, arch, vers))
+	} else if change.IsOrg() && (change.Operation == CHANGE_OPERATION_CREATED || change.Operation == CHANGE_OPERATION_DELETED) {
```

Comment:
There are actually 4 events which can occur where everything in the org cache needs to be deleted. You handle 2 of them here. The other 2 are when the org is added to or removed from the agbot's served list. See this code: https://github.com/open-horizon/anax/blob/master/agreementbot/changes_worker.go#L270